docs/detections/rules-ui-manage.asciidoc

```diff
@@ -104,6 +105,22 @@ NOTE: When duplicating a rule with exceptions, you can choose to duplicate the r
 * To enable or disable a single rule, switch on the rule's *Enabled* toggle.
 * To <<snooze-rule-actions,snooze>> actions for rules, click the bell icon.
 
+[float]
+[[manually-run-rules]]
+=== Manually run rules
+
+Manually run enabled rules during a specfied time range. 
+
+. Go to **Rules** -> **Detection rules (SIEM)**.
+. Do one of the following:
+* Select the **All actions** menu (**...**) on a rule, then select **Manual run**.
+* Select a rule to open its details page, select the **All actions** menu, then select **Manual run**.
```

Comment:
These are very similar steps, maybe combine them? I know they're referencing two different UI areas (the overall rules page and a single rule's details page), but the actual steps are basically the same.

```suggestion
* Select the **All actions** menu on a rule (**...** → **All actions** in the rules list), then select **Manual run**.
```